### ISSUE TITLE
wps pin not found ?

### ISSUE THREAD

**Issue by amazer-bil** [Her0xDa/Her0xDa-Wps-Cracker#1]
Hi brother...i'm asking for a help if that possible :) !

lately i ran reaver and pixiewps and when i did it appeared in the end ' wps pin not found ' and some other rooters it appeared ' warning not associated with this ap ' and stays like that.

thanks :) .

**Comment by Her0xDa** [comment]
Hi sure dude ,  ' warning not associated with this ap ' its appears when the ap is far or ur adapter is weak i prefer using a RT8187L chip 
btw are u using this tool or just reaver folk + pixiewps?

**Comment by amazer-bil** [comment]
yes i have an adapter called ' RT8187 ' from manufacter called ' LineFox ' and the ap is very close to me.actually i have to aps really near me and when i ran reaver and pixiewps.it tooks a little while and then ' warning : failed to associate with this ap ' appears and keep on appearing . 
yea just reaver and pixiewps
 
thanks for responding :)

**Comment by Her0xDa** [comment]
check this cmd before starting
iwconfig wlan0 rate 1M auto

**Comment by amazer-bil** [comment]
when i ran this ' iwconfig wlan0 rate 1m auto ' it gave me nothing
but when i ran ' iwconfig wlan0 ' it gave me ' about access point and mode and bssid and ifee and tx-power and stuff like that .
when i ran reaver and pixiewps same issue .
but i'm not sure if that what you told me to do, i'm new in this stuff and sorry i'm bothering you

**Comment by Her0xDa** [comment]
No problem dude , 
this cmnd configure the default transmission rate in this case u ll have lower  packet loss ratio to avoid 
'failed to associate with this ap ' error indeed when the ap is too far u ll get this , so i guess either u have a fake RT8187 chip or poor drivers 
try reinstall the drivers or use another adapter 
Enjoy !

**Comment by amazer-bil** [comment]
i do have original RT8187 chip but i guess no hope.
thanks alot my friend i appreciate it, keep the good work  :)
